packages/wrapped-keys/src/lib/lit-actions-client/utils.ts

```diff
@@ -48,3 +51,50 @@ export function postLitActionValidation(
 export function getLitActionCid(network: Network, actionType: LitActionType) {
   return LIT_ACTION_CID_REPOSITORY[actionType][network];
 }
+
+export function getLitActionCode(
+  network: Network,
+  actionType: LitActionType
+): string {
+  const litActionCode = LIT_ACTION_BUNDLED_CODE_PATH[actionType][network];
+
+  if (!litActionCode) {
+    throw new Error(
+      `Could not find Lit Action code for action type: ${actionType}`
+    );
+  }
+  return litActionCode;
+}
+
+/**
+ * Fetch the Lit action code or its IPFS CID for a given network and action type.
+ *
+ * @param {Network} network The network to get the code or CID for.
+ * @param {LitActionType} actionType The type of action to get the code or CID for.
+ * @returns {{ litActionCode?: string, litActionIpfsCid?: string }} The Lit action code or its IPFS CID.
+ */
+export function getLitActionCodeOrCid(
```

Comment:
Just want to note for readers that we have introduced a new param in the `executeJs` function where we could overwrite the LitAction `code` property.

https://github.com/LIT-Protocol/js-sdk/blob/816aa6147a3863efa18ba90e0bbf23f0974adae9/packages/lit-node-client-nodejs/src/lib/lit-node-client-nodejs.ts#L1047-L1065

But in this case we are "pulling content" for a static code, so this param is irrelevant.